### PR TITLE
Validation accuracy during training

### PR DESCRIPTION
**Pull request by ramakanth-pasunuru**
- Tracks and plots validation accuracy for 2-way classification tasks during training
- Remove code owners for the opt_instruct branch

### CODE REVIEW

File: metaseq/model_parallel/criterions/vocab_parallel_cross_entropy.py [hunk 1]
```diff
@@ -45,6 +44,7 @@ def forward(self, model, sample, reduce=True):
         loss = vocab_parallel_cross_entropy(net_output[0].float(), target)
         if has_pad:
             loss = loss * (target != self.padding_idx)
+        batch_loss = loss.mean(-1)
```
Comment (stephenroller) [inline metaseq/model_parallel/criterions/vocab_parallel_cross_entropy.py:47]: I don't think we want this. We're going to be dividing by n twice now I believe?

**Review by stephenroller — CHANGES_REQUESTED** [review]
Sorry Red flag

**Comment by sriniiyer** [comment]
Did you do a before-after test? The train ppls are exactly the same before and after this PR, across muiltiple epochs?

**Comment by ramakanth-pasunuru** [comment]
@sriniiyer Thanks for the suggestions. 
1. I checked that the train ppls remain the same for the whole training with and without this PR. 
2. I was able to match the reported accuracy during the training for valid set to be very close to the accuracy I got from gpt3_eval.py --> 47.4474 vs. 47.7477

**Review by sriniiyer — APPROVED** [review]
LGTM - This is awesome!
